hack/update-codegen.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Let's move this file into it's own folder under `tools/` instead of `hack/`.

Reply:
💯